server/bin/www.js

```diff
@@ -0,0 +1,15 @@
+// This will be our application entry. We'll setup our server here.
+// const http = require('http');
```

Comment:
Nice coding practises, however, you could try to avoid pushing code that you don't need in your app